server/tasks/index.ts

```
@@ -0,0 +1,57 @@
+
+    this.agenda.on("start", (job) => {
+      console.log("Job %s starting", job.attrs.name);
+    })
+
+    this.agenda.on("complete", (job) => {
+      console.log("Job %s finished", job.attrs.name);
+    })
+  }
+
+  public static getInstance(): TaskManager {
+    if (!this._instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)